tempest/api/workloadmgr/barbican/test_barbican.py

```diff
@@ -2467,3 +2476,249 @@ def test_6_barbican(self):
                     reporting.set_test_script_status(tvaultconf.FAIL)
                     reporting.add_test_script(test[0])
                     reporting.test_case_to_write()
+            # Delete workload
+            self.workload_delete(self.wid1)
+            # Delete vm
+            self.delete_vm(self.vm_id)
+            LOG.debug("vm deleted succesfully")
+
+            # delete volume
+            self.delete_volume(self.volume_id)
+            LOG.debug("volume deleted successfully")
+
+    # Workload policy with scheduler and retention parameter
+    @test.pre_req({'type': 'barbican_workload'})
+    @decorators.attr(type='workloadmgr_cli')
+    def test_7_barbican(self):
+        reporting.add_test_script(str(__name__) + "_Create_encrypted_workload_with_workload_policy")
+        try:
+            snapshots_list = []
+            global workload_id
```

Comment:
Which workload is getting used here ?

Reply:
encrypted workload is created after policy creation and that is used

Reply:
removed the variable